Makefile

```diff
@@ -0,0 +1,29 @@
+#install kakarot
+
+install:
+	sh kakarotup
+
+# install dependencies, automatically creates a virtual environment
+poetry-install:
+	poetry install
+
+# run devnet
+devnet: poetry-install 
+	 poetry run starknet-devnet --seed 0 --disable-rpc-request-validation --load-path deployments/devnet.pkl --timeout 300
```

Comment:
As per Ivan from Shardlabs, maybe it's better to increase timeout to 5s

> Hello! You may experience timeouts in the devnet (during declare). This would potentially fix the issue https://github.com/0xSpaceShard/starknet-devnet/issues/434 ...Until then, the workaround is to start devnet with a higher timeout, for example --timeout 5000 !

Reply:
done -> [link](https://github.com/sayajin-labs/kakarot-rpc/pull/134/commits/afa06f24336adbf34b6706bbc1ff488aa76eb556)